install/connections.csv

```diff
@@ -739,7 +739,7 @@ RIVH,11,Lathrope Road,Portable B,,2007-03-27T00:00:01Z,2007-05-14T03:40:00Z
 RIVO,10,Riverton,Portable,,2014-03-19T21:30:00Z,2014-09-13T00:00:00Z
 RIZ,10,Raoul Island Met Station,,0,2009-05-18T00:01:00Z,9999-01-01T00:00:00Z
 RIZ,20,Raoul Island Met Station,,3,2009-05-18T01:00:00Z,2016-09-17T13:00:00Z
-RIZ,20,Raoul Island Met Station,Strong Motion,,2016-09-17T13:00:01Z,9999-01-01T00:00:00Z
+RIZ,20,Raoul Island Met Station,Strong Motion,,2016-09-17T13:00:01Z,2023-01-28T21:16:00Z
```

Comment:
Perhaps would  you 
change for :
**"RIZ,20,Raoul Island Met Station,Strong Motion,3,2016-09-17T13:00:01Z,2023-01-28T21:16:00Z"**

just adding the 3 for the pin 3  of the related  datalogger with multiple instruments ...
thats all :)

Reply:
3 added 😄 
another thumbs up/approval please @salichon